.github/scripts/bot-advanced-check.sh

```
@@ -1,93 +1,231 @@
+    -f query='
+      query($owner: String!, $name: String!, $user: String!) {
+        repository(owner: $owner, name: $name) {
+          intermediate: issues(
+            first: 1
+            states: CLOSED
+            filterBy: { assignee: $user, labels: ["intermediate"] }
+          ) {
+            totalCount
+          }
+        }
+      }' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)